<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>461</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>689</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>890</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Collection tool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Downloads/logo.png</normaloff>../../Downloads/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>851</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="project_label">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QTreeWidget" name="all_users_tree">
         <property name="minimumSize">
          <size>
           <width>260</width>
           <height>212</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>420</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="headerCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Имя</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Логин</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTreeWidget" name="collectors_tree">
         <property name="minimumSize">
          <size>
           <width>311</width>
           <height>212</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>420</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="headerHidden">
          <bool>false</bool>
         </property>
         <attribute name="headerVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="headerCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Имя</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Портфель</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QTextEdit" name="search_user">
         <property name="maximumSize">
          <size>
           <width>420</width>
           <height>25</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextEdit" name="search_collector">
         <property name="maximumSize">
          <size>
           <width>420</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="token_btn">
           <property name="text">
            <string>Токен</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="redistribute_btn">
           <property name="text">
            <string>RedistributionOverdueProducts</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="new_portfolio_btn">
           <property name="text">
            <string>AddProductsToNewCollector</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>863</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionReconnect">
   <property name="text">
    <string>Reconnect</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionProjects">
   <property name="text">
    <string>Projects</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
